### PR TITLE
fix: fix typo on debug mode raw diff mode

### PR DESCRIPTION
**Pull request by ytvnr**
Just fix a typo

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/type-debug-diff/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-taeigyxqxc.chromatic.com)

### CODE REVIEW

**Review by phiz71 — APPROVED** [review]

File: gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
       (ngModelChange)="computeDiff()"
       aria-label="Output format button group"
     >
-      <mat-button-toggle value="row">Row</mat-button-toggle>
+      <mat-button-toggle value="row">Raw</mat-button-toggle>
```
Comment (mouligno) [inline gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html]: shouldn't we change the value also ?

Comment (phiz71) [reply gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html]: good catch ;)

Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html]: 👍🏻  and also fix `line-by-lin` a few lines below

Comment (ytvnr) [reply gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html]: good catch indeed, and I have an old commit 🗡️

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/shared/components/gio-diff/gio-diff.component.html]: 🙇

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**